apprise_api/api/templates/config.html

```diff
@@ -71,7 +71,7 @@ <h5>{% trans "Loaded Configuration" %}</h5>
   <div id="config" class="col s12">
     <p>
       {% blocktrans %}Define your configuration below:{% endblocktrans %}
-      <form id="addconfig" action="{% url "add" key %}" method="post">
```

Comment:
I think this entry works perfectly fine, no?  I would think that using the django reverse url lookup (where it is possible) is better then hard-coding the URL?